tests/queries/0_stateless/02134_async_inserts_formats.sh

```diff
@@ -4,7 +4,7 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-url="${CLICKHOUSE_URL}&async_insert=1&wait_for_async_insert=1"
+url="${CLICKHOUSE_URL}&async_insert=1&wait_for_async_insert=1&async_insert_use_adaptive_busy_timeout=0"
```

Comment:
why do you have to disable it here?

Reply:
The test became flaky after enabling adaptive timeouts by default:
https://pastila.nl/?01884edd/3302a987b576ef7e75fbc3d8e1e0bf3d#Dgf6yOJGn0Bk35nJDt0++g==

I suspect this is because a queue can be flushed synchronously, depending on how much time has elapsed since the last insert.

Reply:
> I suspect this is because a queue can be flushed synchronously

yes, that makes sense

Reply:
Thanks for the review. I'll wait for the current workflow to end, then modify the test.

Reply:
Same for `async_insert_dedup_replicated_collapsing.sh` test: duplicate entries are flushed in separate batches if the timeout is small. So I disabled the feature for that test.